test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -78,7 +71,11 @@ test.describe(
 			await editor.insertBlock( { name: 'core/navigation' } );
 
 			// Check the block in the canvas.
-			await navBlockUtils.selectNavigationItemOnCanvas( 'WordPress' );
+			await expect(
+				this.editor.canvas.locator(
```

Comment:
This should just be `editor`.

```suggestion
				editor.canvas.locator(
```